lib/rubocop/config.rb

```diff
@@ -295,6 +296,15 @@ def target_ruby_version
         end
     end
 
+    def target_rails_version
+      @target_rails_version ||=
+        if for_all_cops['TargetRailsVersion']
```

Comment:
Could possibly be:

```ruby
for_all_cops['TargetRailsVersion'] || DEFAULT_RAILS_VERSION
```

🙂 

I also thought this would be a `Style/MultilineMemoization` offense ... might be a false negative? 🤔